versioned/persist/mongodb/src/test/java/org/projectnessie/versioned/persist/mongodb/FlapdoodleMongoTestConnectionProviderSource.java

```diff
@@ -28,59 +28,82 @@
 import de.flapdoodle.embed.process.config.io.ProcessOutput;
 import de.flapdoodle.embed.process.io.StreamProcessor;
 import de.flapdoodle.embed.process.runtime.Network;
-import java.util.concurrent.atomic.AtomicInteger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.assertj.core.util.VisibleForTesting;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * MongoDB test connection-provider source using a locally spawned MongoDB instance via
  * "flapdoodle".
  */
 public class FlapdoodleMongoTestConnectionProviderSource extends MongoTestConnectionProviderSource {
-  private static final Pattern LISTEN_ON_PORT_PATTERN =
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(FlapdoodleMongoTestConnectionProviderSource.class);
+
+  @VisibleForTesting
+  static final Pattern LISTEN_ON_PORT_PATTERN =
       Pattern.compile(
-          ".*NETWORK ([^\\n]*) waiting for connections on port ([0-9]+)\\n.*",
+          ".*NETWORK.*[Ww]aiting for connections[^\n]*port[^\n0-9]*([0-9]{2,})[^0-9].*",
```

Comment:
nit: use a case-insensitive RegEx perhaps?... but I'm +1 on this change in general... no need to adjust ATM.